credential/zkpass/v0.1.0/schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/credential/zkpass/v0.1.0/schema.json",
+    "title": "Schema for ZKPass",
+    "description": "A schema for zkpass credential",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "database": {
+        "name": "zkpass_credentials",
+        "indexes": {
+            "taskId": ["taskId"]
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "https://common.schemas.verida.io/core/base/schema.json"
+        },
+        {
+            "properties": {
+                "taskId": {
+                    "title": "Task ID",
+                    "description": "Unique id of the task allocated by the allocator node",
+                    "type": "string"
+                },
+                "publicFields": {
+                    "title": "ZkPass PublicFields",
+                    "description": "Values of public fields defined in schema",
+                    "type": "array",
+                    "items": {
+                        "type": "object"
```

Comment:
Is there an example of what these values are?

I'm not sure if they are an array of `object` or `string`?

Reply:
Yes, you are right. It is `string`, as it's value of public fields from zkPass schema.
They support <, >, = operations for public fields, so it should be `string`, not object.